src/Http/Http.Extensions/src/RequestDelegateFactory.cs

```diff
@@ -22,6 +22,8 @@ namespace Microsoft.AspNetCore.Http
     /// </summary>
     public static partial class RequestDelegateFactory
     {
+        private static readonly NullabilityInfoContext nullabilityContext = new NullabilityInfoContext();
```

Comment:
Nit:

```suggestion
        private static readonly NullabilityInfoContext NullabilityContext = new NullabilityInfoContext();
```

Or `_nullabilityContext` is also fine.

I wonder if we should clear both this and the TryParseMethodCache when build is complete, but that's not related to this PR.

Reply:
I think we would make an application and take a memory dump and see how much memory is being used up and determine based on that.